TimeFormat.java

```diff
@@ -12,6 +12,23 @@ public static void main(String[] args) {
 		int hours = Integer.parseInt("" + args[0].charAt(0) + args[0].charAt(1));
 		// Does the same with the minutes part of the input.
 		int minutes = Integer.parseInt("" + args[0].charAt(3) + args[0].charAt(4));
-        // Replace this comment with the rest of your code
+		if (hours>12){
+			if (minutes<10){
+				System.out.println( hours-12 + ":0" + minutes + " PM");
+			}
+			else
+				System.out.println(hours-12 + ":" + minutes + " PM" );	
+		}
+		if (hours<12) {

```

Comment:
Just a thought: is the usage of `else if` and then in the end `else` would have change the logic and the functionality?